cloud/scope/powervs_machine.go

```diff
@@ -1060,6 +1060,35 @@ func (m *PowerVSMachineScope) CreateVPCLoadBalancerPoolMember() (*vpcv1.LoadBala
 					}
 				}
 			}
+
+			for _, al := range lb.AdditionalListeners {
+				var shouldAdd bool
+
+				machineType, ok := m.IBMPowerVSMachine.ObjectMeta.Labels["powervs.cluster.x-k8s.io/node-type"]
```

Comment:
Should we use  label or annotation for selecting the machines to configure, Usually CAPI uses annotation to select the machines.
https://github.com/Karthik-K-N/cluster-api/blob/f3d39741b47ae458625a28e6b37e3adeb18dfeb1/api/v1beta1/common_types.go#L98-L103